index.js

```diff
@@ -63,6 +63,7 @@ var slogans = [
     "— Qualitysoftware since 1784",
     "- flieg und sieg!",
     "- Go! Go! Go!"
```

Comment:
Misses an excellent `,` :) Otherwise excellent addition to our growing list of capabilities.

Reply:
oha! fixed!